airflow/contrib/auth/backends/ldap_auth.py

```diff
@@ -55,16 +55,18 @@ class LdapException(Exception):
 
 
 def get_ldap_connection(dn=None, password=None):
-    tls_configuration = None
-    use_ssl = False
     try:
         cacert = configuration.conf.get("ldap", "cacert")
-        tls_configuration = Tls(validate=ssl.CERT_REQUIRED, ca_certs_file=cacert)
-        use_ssl = True
-    except Exception:
+    except AirflowConfigException:
         pass
```

Comment:
Minor Note:
Can we add a note here in `except` block saying that "Cacert is missing and it is now mandatory"? rather than just a missing Airflow config?

What do you think?

Reply:
It's not required - if it is not specified then it will use the normal system cert chain to verify the LDAP server cert.

This is only required if a self-signed/private-CA is used.

Reply:
(see note in UPDATING.md - it mentions this)

Reply:
Oh yes, makes sense.